internal/reader/home_chain_test.go

```diff
@@ -175,7 +175,7 @@ func Test_PollingWorking(t *testing.T) {
 		}
 	}
 	// called at least 2 times, one for start and one for the first tick
-	require.GreaterOrEqual(t, callCount, 2)
+	require.GreaterOrEqual(t, callCount, 1)
```

Comment:
Actually this will still be 2.

Reply:
the number of calls should still be 2 as it will poll as soon as it's started and then again when it starts polling.
If it's not then something is not correct in the polling itself.

Reply:
yeap, updated to 2